rlclientlib/live_model_impl.h

```diff
@@ -87,6 +90,7 @@ namespace reinforcement_learning
     std::unique_ptr<model_management::i_data_transport> _transport{nullptr};
     std::unique_ptr<model_management::i_model> _model{nullptr};
     std::unique_ptr<logger::cb_logger_facade> _ranking_logger{nullptr};
+    std::unique_ptr<logger::ca_logger_facade> _continuous_action_logger{nullptr};
```

Comment:
Not sure if want to have it in this PR, but:
Since we are working on new loggers which are not having legacy implementation like ca and multistep (and v2 is unified), we probably can refactor this code like this:
class interaction_logger_facade{
...
const std::unique_ptr<interaction_logger> _v1_cb;
const std::unique_ptr<ccb_logger> _v1_ccb;
...
const std::unique_ptr<generic_logger> _v2;
}
It will at reduce copy-paste around v2 + localize legacy code inside logger_facade v1 branches.

Reply:
Supporting CA under v1 will be harder than needed.

Reply:
I can look at this in a separate PR once the loggers are a bit more stabilized (regarding https://github.com/VowpalWabbit/reinforcement_learning/pull/192#discussion_r496224392)

Reply:
sure, sounds good.